spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/outbound/AbstractMqttMessageHandler.java

```diff
@@ -86,12 +90,19 @@ public abstract class AbstractMqttMessageHandler extends AbstractMessageHandler
 
 	private int clientInstance;
 
+	protected ClientManager<T> clientManager;
```

Comment:
Same here: `private final`.
We really still believe that `final` provides some performance optimization since the value of the property never changes.